docs/manual/chapters/schemas.qmd

```diff
@@ -133,13 +133,17 @@ the filter argument is:
 **column name** **operator** **value**
 ```
 
-Where the "column name" the name of a column (specified in the `key` column)
-from the table(s) specified in the `table` column, and the "value" is the value
-or range of values that determine whether a row is selected for sharing, stored
-in the `value` column. The "operator" is a placeholder for the symbol indicates
-that the nature of the filter to be applied, and the desired relationship
-between the `key` and the `value`. The currently accepted values for the
-`operator` column are:
+"column name" is the name of a column (specified in the `key` column) from the
+table(s) specified in the `table` column.
+
+"value" is the value, or range of values, that determine whether a row is
+selected for sharing. It's stored in the `value` column. String values can
+optionally be quoted with double-quotes (`"`) to allow commas, while
```

Comment:
Would it be better to allow them to escape commas using a backslash? This way single quotes are also allowed in the string values.

Reply:
it's a csv parsing standard that quotes can be used to escape the delimiter character, so I think we should keep it that way, and it's not part of our code either. I mainly documented it because I experienced that double and single quotes work differently.

Reply:
oh, I'm sorry, I misunderstood. Single quotes can already be in the value, that's what I was trying to document, that they are different from double quotes, so this would work: `"john's table"`